orm/genesis.go

```diff
@@ -1,16 +1,18 @@
 package orm
 
 import (
-	"reflect"
-
-	"github.com/cosmos/cosmos-sdk/store/prefix"
 	"github.com/cosmos/cosmos-sdk/types/errors"
 )
 
 // TableExportable
 type TableExportable interface {
-	// Table returns the table to export
-	Table() table
+	// ExportIterator returns an iterator over the values to export
+	ExportIterator(HasKVStore) (Iterator, error)
```

Comment:
I'm wondering if we should actually make this `TableExportable` interface more generic, that all table types would implement with just `Export(ctx HasKVStore, dest ModelSlicePtr) (uint64, error)` and `Import(ctx HasKVStore, data interface{}, seqValue uint64) error` methods